index.html

```diff
@@ -1,63 +1,73 @@
-      rel="stylesheet"
-    />
-    <link rel="stylesheet" href="./assets/css/style.css" />
-  </head>
 
-  <body>
-    <header>
-      <nav class="nav-bar">
-        <a href="#"
-          ><img src="./assets/img/ic_menu.png" alt="Hamburguer menu icon"
-        /></a>
-      </nav>
-    </header>
-    <main>
-      <section class="headline">
-        <div class="text-wrapper">
-          <h1>Felipe<br />Oyarzo</h1>
-          <p>
-            Hello! I am a software developer! I can help you build a product,
-            feature or website. Take a look of my works. If you like what you
-            see and have a project you need coded, don't hesitate and contact
-            me.
-          </p>
-        </div>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="author" content="Felipe Oyarzo" />
+  <title>My Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@300;400;500;700&display=swap"
+    rel="stylesheet" />
+  <link rel="stylesheet" href="./assets/css/style.css" />
+</head>
+
+<body>
+  <header>
+    <nav class="nav-bar">
+      <div id="menu-wrapper">
+        <a href="#">Hello</a>
+        <a href="#">Portfolio</a>
+        <a href="#">about</a>
+        <a href="#">Contact</a>
+      </div>
+      <a href="#"><img src="./assets/img/ic_menu.png" alt="Hamburguer menu icon" id="hamburguer" /></a>
+    </nav>
+  </header>
+  <main>
+    <section class="headline">
+      <div class="text-wrapper">
+        <h1>Felipe<br />Oyarzo</h1>
+        <p>
+          Hello! I am a software developer! I can help you build a product,
+          feature or website. Take a look of my works. If you like what you
+          see and have a project you need coded, don't hesitate and contact
+          me.
+        </p>
         <button class="button" type="button" id="btnHeadline">
           Start collaboration
         </button>
-      </section>
-      <section class="projects">
-        <h1>Projects</h1>
-        <div class="grid-container">
-          <div class="card card1">
-            <div class="card-image image1"></div>
-            <div class="card-body">
-              <p class="project-title">Project name goes here</p>
-              <ul class="card-list">
-                <li class="card-list-item">HTML/CSS</li>
-                <li class="card-list-item">Ruby on Rails</li>
-                <li class="card-list-item">JavaScript</li>
-              </ul>
-              <button type="button" class="card-btn">
-                See this project &rarr;
-              </button>
-            </div>
+      </div>
+    </section>
+    <section class="projects">
+      <h2>Projects</h2>
```

Comment:
- [ ] This header is missing from your design, kindly fix this issue 👍 
## Your design
![image](https://user-images.githubusercontent.com/50721479/187006074-196aae95-aea5-4bb9-9664-e01839d4f404.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/187006043-409dec57-8750-48ac-b314-cb32b9d51a93.png)

Reply:
Hello @ha-manel , thank you very much for your feedback. On my computer, it shows correctly, what could be the cause of this? maybe browser compatibility with the transform property? I would really appreciate it if you could give me a hint about this one because I really don't know why it isn't showing on your side 😢 
![image](https://user-images.githubusercontent.com/101157253/187006821-6a285bf4-cd31-4075-90b6-12af331ce54a.png)

Reply:
hey @jfoyarzo, to be honest, I'm not sure what's the issue! but since it's showing on your side I'll discard this comment 😊 great job keep it up 👍

Reply:
@ha-manel Thank you very much 😸